inference-engine/src/mkldnn_plugin/nodes/mkldnn_deconv_node.cpp

```diff
@@ -222,12 +222,13 @@ void MKLDNNDeconvolutionNode::getSupportedDescriptors() {
         IE_THROW() << errorPrefix << " has incorrect number of output edges";
 
     for (int i = 0; i < paddingR.size(); i++) {
+        if (paddingR[i] != 0) continue;
```

Comment:
I think we decided to remove this code block (which recomputates paddingR) at all?

Reply:
I removed this code block in 238baefc5bf89c6f01275c67985c1369ec502523, but I got a similar error for the nnart_001 network. I think this code block is needed if paddingR = 0

Reply:
I cannot agree with such conclusion. It looks like WA. Most likely decision should we recompute padding or not depends on `auto_pad` attribute: https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/convolution/ConvolutionBackpropData_1.md. Please continue investigation

Reply:
@dmitry-gorokhov The problem was wrong division. Fixed in 954579003a3d3c8937c03ed6c879ffb290fe438f